packaging/examples/metrics/kafka-metrics.yaml

```diff
@@ -65,6 +65,8 @@ spec:
   kafkaExporter:
     topicRegex: ".*"
     groupRegex: ".*"
+    topicExcludeRegex: "^$"
+    groupExcludeRegex: "^$"
```

Comment:
I'm not sure we want to include these in the examples as they have rather secondary meaning ... what do you think @ppatierno?

Reply:
Yes I agree.